rskj-core/src/test/java/co/rsk/peg/InMemoryStorageSimulator.java

```diff
@@ -0,0 +1,41 @@
+package co.rsk.peg;
+
+import co.rsk.peg.storage.StorageAccessException;
+import co.rsk.peg.storage.StorageAccessor;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Objects;
+import org.ethereum.vm.DataWord;
+
+/**
+ * This is an In-Memory Storage Simulator to be used in Unit Tests to reduce the use of mocks, thus
+ * improving the Unit Tests reliability.
+ */
+public class InMemoryStorageSimulator implements StorageAccessor {
```

Comment:
Why Simulator? Nothing simulated the way I see it, just an storage in memory

Reply:
Agree with Marcos. As said in the thread, this is In Memory Storage, nothing is being simulated here.

Reply:
Hey, guys!
There's no problem about changing the name to InMemoryStorage.
About conceptually talking, I have a different standpoint. I stand by my vision of this since if we speak about storage as a component, it needs to have some quality attributes and an architecture that a simple data structure doesn't have.

Reply:
Done! Renamed the class from InMemoryStorageSimulator to InMemoryStorage.